SwrveSDK/src/google/java/com/swrve/sdk/config/SwrveConfig.java

```diff
@@ -62,4 +67,18 @@ public boolean isGAIDLoggingEnabled() {
     public void setGAIDLoggingEnabled(boolean enabled) {
         this.gAIDLoggingEnabled = enabled;
     }
+
+    /**
+     * @return if it will automatically obtain the Google Cloud Messaging Registration ID.
+     */
+    public boolean isObtainRegistrationIdEnabled() {
```

Comment:
We need a way for our SDK to stop asking for a registration id. Otherwise it might get a different value / clash with the app asking for it.

We've had a report that multiple queries to the GCM API to get the reg id can cause invalid values.